config/locales/simple_form.fa.yml

```diff
@@ -4,14 +4,20 @@ fa:
     hints:
       defaults:
         avatar: یکی از قالب‌های PNG یا  GIF یا JPG. بیشترین اندازه ۲ مگابایت. تصویر به اندازهٔ ۱۲۰×۱۲۰ پیکسل تبدیل خواهد شد.
-        display_name: '%{count} حرف باقی مانده'
+        display_name:
+            one: '<span class="name-counter">1</span> حرف باقی مانده'
```

Comment:
the whitespace indent difference between display_name and one/other is 4 spaces instead of 2. this breaks the syntax. see also: same for "note" below